addons/binding/org.openhab.binding.solarlog/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solarlog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="datalogger">
+		<label>SolarLog Binding Thing</label>
+		<description>Sample thing for SolarLog Binding</description>
+
+		<channels>
+			<channel id="lastupdate" typeId="lastupdate"></channel>
+			<channel id="pac" typeId="pac"></channel>
+			<channel id="pdc" typeId="pdc"></channel>
+			<channel id="uac" typeId="uac"></channel>
+			<channel id="udc" typeId="udc"></channel>
+			<channel id="yieldday" typeId="yieldday"></channel>
+			<channel id="yieldyesterday" typeId="yieldyesterday"></channel>
+			<channel id="yieldmonth" typeId="yieldmonth"></channel>
+			<channel id="yieldyear" typeId="yieldyear"></channel>
+			<channel id="yieldtotal" typeId="yieldtotal"></channel>
+			<channel id="conspac" typeId="conspac"></channel>
+			<channel id="consyieldday" typeId="consyieldday"></channel>
+			<channel id="consyieldyesterday" typeId="consyieldyesterday"></channel>
+			<channel id="consyieldmonth" typeId="consyieldmonth"></channel>
+			<channel id="consyieldyear" typeId="consyieldyear"></channel>
+			<channel id="consyieldtotal" typeId="consyieldtotal"></channel>
+			<channel id="totalpower" typeId="totalpower"></channel>
+		</channels>
+		<config-description>
+			<parameter name="url" type="text" required="true">
+				<label>URL</label>
+				<description>URL of the SolarLog web interface (e.g. http://solarlog)</description>
+				<default>http://solar-log</default>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" min="15">
+				<label>Refresh Interval</label>
+				<description>States how often a refresh shall occur (in s). SolarLog Updates Data all 15s</description>
+				<default>15</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="lastupdate">
+		<item-type>DateTime</item-type>
+		<label>Last Update Time</label>
+		<description>Last Update Time</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="pac">
+		<item-type>Number</item-type>
+		<label>Pac</label>
+		<description>Total output PAC from all of the inverters</description>
+		<state readOnly="true" pattern="%d W"></state>
```

Comment:
Are you sure that all of your channel states are always only integers? Otherwise, better use a "%.1f W" pattern or similar.